### PR TITLE
updated .gitignore, established server, connection to API

### PR DESCRIPTION
**Pull request by dirtOval**
Simple proof of concept for connecting to API thru express server. Didn't run into any issues with typescript, which is a little sinister/concerning.

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -41,6 +41,7 @@ nav {
   height: 55rem;
   display: flex;
   flex-direction: column;
+  transition: font-size 1s ease;
```
Comment (abanderson3) [inline src/styles/main.scss]: Should this transition still be here?

**Review by abanderson3 — COMMENTED** [review]
I had one comment, it looks like the there is still a transition in the scss file.

File: src/styles/main.scss (hunk 1 above)
Comment (dirtOval) [reply src/styles/main.scss]: no . . . 🥺👉👈

**Review by abanderson3 — APPROVED** [review]
Thanks for taking care of that.

**Pull request merged**